sdk/storagecache/Azure.ResourceManager.StorageCache/src/autorest.md

```diff
@@ -0,0 +1,64 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: StorageCache
+namespace: Azure.ResourceManager.StorageCache
+require: https://github.com/Azure/azure-rest-api-specs/blob/53b1affe357b3bfbb53721d0a2002382a046d3b0/specification/storagecache/resource-manager/readme.md
+tag: package-2022-05
+output-folder: $(this-folder)/Generated
+clear-output-folder: true
+skip-csproj: true
+modelerfour:
+  flatten-payloads: false
+
+ 
+
+format-by-name-rules:
+  'tenantId': 'uuid'
+  'ETag': 'etag'
+  'location': 'azure-location'
+  '*Uri': 'Uri'
+  '*Uris': 'Uri'
+
+rename-rules:
+  CPU: Cpu
+  CPUs: Cpus
+  Os: OS
+  Ip: IP
+  Ips: IPs|ips
+  ID: Id
+  IDs: Ids
+  VM: Vm
+  VMs: Vms
+  Vmos: VmOS
+  VMScaleSet: VmScaleSet
+  DNS: Dns
+  VPN: Vpn
+  NAT: Nat
+  WAN: Wan
+  Ipv4: IPv4|ipv4
+  Ipv6: IPv6|ipv6
+  Ipsec: IPsec|ipsec
+  SSO: Sso
+  URI: Uri
+  Etag: ETag|etag
+
+directive:
+  - rename-operation:
+      from: StorageTarget_Flush
+      to: StorageTargets_Flush
+  - rename-operation:
+      from: StorageTarget_Suspend
+      to: StorageTargets_Suspend
+  - rename-operation:
+      from: StorageTarget_Resume
+      to: StorageTargets_Resume
+  - rename-operation:
+      from: StorageTarget_Invalidate
+      to: StorageTargets_Invalidate
```

Comment:
I am OK with the renamings here, but why do we have to do this?
As far as I know, even if we do not have these renamings, the generation should be fine. But this will change the information in our operations' summary to show the incorrect operationId.
If possible we could remove this..